rest_framework/fields.py

```diff
@@ -1105,7 +1106,10 @@ def enforce_timezone(self, value):
         field_timezone = getattr(self, 'timezone', self.default_timezone())
 
         if (field_timezone is not None) and not timezone.is_aware(value):
-            return timezone.make_aware(value, field_timezone)
+            try:
+                return timezone.make_aware(value, field_timezone)
+            except Exception:
```

Comment:
Without pytz I can't get timezone `pytz.timezone('America/New_York')` object, so if pytz is not installed I got: `AttributeError: 'NoneType' object has no attribute 'timezone'`.
Can you please point me on that compact model, I'll give it a try.